content/en/docs/tasks/security/authentication/mtls-migration/index.md

```diff
@@ -7,97 +7,32 @@ aliases:
   learn how to configure authentication policy.
 
-* Have a Kubernetes cluster with Istio installed, without global mutual TLS enabled (e.g use the demo configuration profile as described in
-[installation steps](/docs/setup/getting-started), or set the `global.mtls.enabled` installation option to false).
-
-* Ensure that your cluster is in `PERMISSIVE` mode before migrating to mutual TLS.
-  Run the following command to check:
-
-    {{< text bash >}}
-    $ kubectl get meshpolicy default -o yaml
-    ...
-    spec:
-      peers:
-      - mtls:
-          mode: PERMISSIVE
-    {{< /text >}}
-
-    If the output is the same as above, you can skip the next step.
-
-* Run the following command to enable `PERMISSIVE` mode for the cluster. In general, this operation
-  does not cause any interruption to your workloads, but note the warning message below.
-
-    {{< warning >}}
-    In `PERMISSIVE` mode, the Envoy sidecar relies on the
-    [ALPN](https://en.wikipedia.org/wiki/Application-Layer_Protocol_Negotiation) value `istio` to
-    decide whether to terminate the mutual TLS traffic. If your workloads (without Envoy sidecar)
-    have enabled mutual TLS directly to the services with Envoy sidecars, enabling `PERMISSIVE` mode
-    may cause these connections to fail.
-    {{< /warning >}}
-
-    {{< text bash >}}
-    $ kubectl apply -f - <<EOF
-    apiVersion: "authentication.istio.io/v1alpha1"
-    kind: "MeshPolicy"
-    metadata:
-      name: "default"
-    spec:
-      peers:
-      - mtls:
-          mode: PERMISSIVE
-    EOF
-    {{< /text >}}
-
-{{< tip >}}
-You can use the following command to show the mutual TLS configuration of all connections from one pod:
-
-{{< text bash >}}
-$ istioctl authn tls-check <YOUR_POD> -n <YOUR_NAMESPACE>
-{{< /text >}}
-
-{{< /tip >}}
-
-The rest of this task is divided into two parts.
-
-* If you want to enable mutual TLS for your workloads one by one, refer to
-[gradually enable mutual TLS for services](/docs/tasks/security/authentication/mtls-migration/#option-1-gradually-enable-mutual-tls-for-services),
-which describes the process using simple examples.
-
-* If you want to enable mutual TLS for the entire cluster, refer to
-[globally enable mutual TLS for the cluster](/docs/tasks/security/authentication/mtls-migration/#option-2-globally-enable-mutual-tls-for-the-cluster).
-
-## Option 1: gradually enable mutual TLS for services
+* Have a Kubernetes cluster with Istio installed, without global mutual TLS enabled (e.g use the demo configuration profile as described in [installation steps](/docs/setup/getting-started).
 
 In this section, you can try out the migration process by creating sample workloads and modifying
 the policies to enforce STRICT mutual TLS between the workloads.
 
-### Set up the cluster
+## Set up the cluster
 
 * Create the following namespaces and deploy [httpbin]({{< github_tree >}}/samples/httpbin) and [sleep]({{< github_tree >}}/samples/sleep) with sidecars on both of them.
```

Comment:
I would suggest changing lines 37-53 to the following:
```
* Create two namespaces, `foo` and `bar`, and deploy [httpbin]({{< github_tree >}}/samples/httpbin) and [sleep]({{< github_tree >}}/samples/sleep) with sidecars on both of them:

    {{< text bash >}}
    $ kubectl create ns foo
    $ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n foo
    $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n foo
    $ kubectl create ns bar
    $ kubectl apply -f <(istioctl kube-inject -f @samples/httpbin/httpbin.yaml@) -n bar
    $ kubectl apply -f <(istioctl kube-inject -f @samples/sleep/sleep.yaml@) -n bar
    {{< /text >}}

* Create another namespace, `legacy`, and deploy [sleep]({{< github_tree >}}/samples/sleep) without a sidecar:

    {{< text bash >}}
    $ kubectl create ns legacy
    $ kubectl apply -f @samples/sleep/sleep.yaml@ -n legacy
    {{< /text >}}
```

Reply:
done.